packages/js/tracking-jsdoc/publish.js

```diff
@@ -0,0 +1,132 @@
+const fs = require( 'jsdoc/fs' );
+const env = require( 'jsdoc/env' );
+const path = require( 'jsdoc/path' );
+
```

Comment:
In this comment https://github.com/woocommerce/grow/pull/1#issuecomment-1131903460

It was suggested to use ESModules syntax but we are using here commonJS

Reply:
I suggested to use ESModules, to keep our code as up to date with the latest syntax aspossible.

Unfortunately, `jsdoc` does not support plugins written as modules, as its plugin framework relies on synchronous CJS `require`. And fixing that in `jsodc` would be quite a big architectural PR.

Reply:
I'd love to use ESM here. I tried making that change to jsdoc as looking for `jsdoc` alternatives but didn't find anything feasible.